P/PETSc/build_tarballs.jl

```diff
@@ -80,8 +88,8 @@ else
 fi
 
 atomic_patch -p1 $WORKSPACE/srcdir/patches/mingw-version.patch
-atomic_patch -p1 $WORKSPACE/srcdir/patches/mpi-constants.patch
-atomic_patch -p1 $WORKSPACE/srcdir/patches/sosuffix.patch   
+atomic_patch -p1 $WORKSPACE/srcdir/patches/mpi-constants.patch     
```

Comment:
```suggestion
atomic_patch -p1 $WORKSPACE/srcdir/patches/mpi-constants.patch
```

Reply:
we do need the `sosuffix` patch. I'vd removed the trailing space after the `mpi-constants.patch`